packages/vc-verification/src/resolver/Ipfs-credential-resolver.ts

```diff
@@ -108,11 +108,13 @@ export class IpfsCredentialResolver implements CredentialResolver {
     namespace: string
   ): Promise<RoleEIP191JWT | undefined> {
     const eip191Jwts = await this.eip191JwtsOf(did);
-    return eip191Jwts.find(
-      (jwt) =>
-        jwt?.payload?.claimData.claimType === namespace ||
-        utils.namehash(jwt?.payload?.claimData.claimType) === namespace
-    );
+    return eip191Jwts
+      .filter((jwt) => typeof jwt.payload.claimData.claimType === 'string')
```

Comment:
Would it make sense to use this helper function here? https://github.com/energywebfoundation/ew-credentials/blob/7475c9a77d5ead021ef5069a62ca315764598270/packages/vc-verification/src/utils/formatters.ts#L96

Reply:
The `isEIP191Jwt` method doesn't validate the case which Dmitry is trying to check. Maybe we make some modifications to `isEIP191Jwt` and then use it here.

Reply:
> Maybe we make some modifications to isEIP191Jwt and then use it here.

What do you think of this approach? To me this would make sense because I think this new line is just an additional check to determine if it is an `EIP191Jwt` or not

Reply:
I think we need dome ultimate solution. I've gave a try to ts-interface-checker/ts-interface-builder. This time I generated types manually, but we can easy automate this. What do think of this approach?